docs/src/pages/components/buttons/ButtonSizes.js

```diff
@@ -45,6 +45,12 @@ export default function ButtonSizes() {
         <IconButton aria-label="delete">
           <DeleteIcon />
         </IconButton>
+        <IconButton aria-label="delete" size="large">
```

Comment:
I would replace the components using `fontSize` with the new ones. It brings confusion as to what is the correct usage.

Reply:
This demo is too crowded. How about we move the IconButton size under https://deploy-preview-26748--material-ui.netlify.app/components/buttons/#icon-button as a new `### Size` header, while retaining inline previews?